pallets/crowdloan-claim/README.md

```diff
@@ -0,0 +1,122 @@
+# Crowdloan Claim Pallet
+
+<!-- TOC -->
+
+- [Crowdloan Claim Pallet](#crowdloan-claim-pallet)
+    - [Overview](#overview)
+    - [Pallet Usage](#pallet-usage)
+        - [Add the Pallet to your Parachain Runtime](#add-the-pallet-to-your-parachain-runtime)
+        - [Configure the Pallet](#configure-the-pallet)
+    - [Pallet Dependencies](#pallet-dependencies)
+    - [Pallet Interface](#pallet-interface)
+        - [Types Declaration](#types-declaration)
+        - [Dispatchable Functions](#dispatchable-functions)
+        - [Pallet Errors](#pallet-errors)
+    - [Pallet Documentation](#pallet-documentation)
+    - [References](#references)
+    - [License](#license)
+
+<!-- /TOC -->
+
+## Overview
+
+This pallet (or module) provides functionalities for rewarding contributors
+who forgo staking on a Polkadot/Kusama relay chain so that to participate in
+a commmunity-funding campaign, called [crowdloan](https://github.com/paritytech/polkadot/blob/master/runtime/common/src/crowdloan.rs),
+for acquiring a parachain slot on the relay chain. For doing so, contributors 
+lock DOTs on the Polkadot/Kusama relay chain, for the duration of the parachain 
+slot auction compaign. If the campaign is successfully closed, for forgoing staking 
+during the crowdloan campaign, contributors are rewarded in native tokens of the 
+parachain (on a parachain account). Otherwise, if the campaign miserably fail, the
+locked DOTs are given back to the contributor.
+
+This rather "generic" Claim Module` acts as a proxy between the contributor
+who claims a reward payout and the `Reward Pallet`, that concretely implement
+the rewarding mechanism, using vesting or not, for instance. The two pallets
+are loosely-coupled by means of the `RewardMechanism` trait (see Configure 
+the Pallet below).
+
+## Pallet Usage
+
+### Add the Pallet to your Parachain Runtime
+
+In order to add this pallet to your runtime, you should add the following lines
+to your parachain's main `Cargo.toml` file:
+
+```toml
+# -- snip --
+
+[dependencies.pallet-crowdloan-claim]            # <-- Add the new dependency
+default_features = false
+git = 'https://github.com/centrifuge-chain/pallet-crowdloan-claim.git'
+branch = master
```

Comment:
To be honest, I am not the biggest fan of this kind of readmes as they get outdated really soon. Did you generate them or do we need to manually adjust all the time? In case of the second, I would prefer to remove them and solely use the docs in the files.